test/e2e_node/services/services.go

```diff
@@ -381,11 +382,12 @@ func (es *e2eService) startKubeletServer() (*server, error) {
 		"--serialize-image-pulls", "false",
 		"--config", framework.TestContext.ManifestPath,
 		"--file-check-frequency", "10s", // Check file frequently so tests won't wait too long
-		"--v", LOG_VERBOSITY_LEVEL, "--logtostderr",
 		"--pod-cidr=10.180.0.0/24", // Assign a fixed CIDR to the node because there is no node controller.
 		"--eviction-hard", framework.TestContext.EvictionHard,
 		"--eviction-pressure-transition-period", "30s",
 		"--feature-gates", framework.TestContext.FeatureGates,
+		"--experimental-runtime-integration-type", framework.TestContext.RuntimeIntegrationType, // Whether to use experimental cri integration.
```

Comment:
We could append the flag only if `framework.TestContext.RuntimeIntegrationType` is not empty